lib/mockzilla-management-ui/common/src/commonMain/kotlin/com/apadmi/mockzilla/desktop/engine/device/ActiveDeviceManager.kt

```diff
@@ -0,0 +1,93 @@
+package com.apadmi.mockzilla.desktop.engine.device
+
+import com.apadmi.mockzilla.lib.models.MetaData
+
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.SharedFlow
+import kotlinx.coroutines.launch
+
+interface ActiveDeviceMonitor {
```

Comment:
should this be called `Monitor` if the file and implementing class are called `Manager`?